core/src/main/java/com/dooapp/fxform/model/impl/AbstractFieldElement.java

```diff
@@ -31,6 +39,33 @@
 	public AbstractFieldElement(Field field) throws FormException {
 		super();
 		this.field = field;
+		init();
+	}
+
+	protected void init() {

```

Comment:
Why did you move this method from AbstractSourceElement to this class ? I think that the init() implementation should be still in AbstractSourceElement.

Reply:
We must ensure the field is set before the creation of the wrappedProperty, needed to build the right type of binding.
By adding the init implementation in the parent constructor the field will still be null.

So another solution would be to manually call the init method after the full object is built but it will many changes in the project as well as break cause breaking change to the current users. This is a thing we don't actually want.